server/src/main/java/org/elasticsearch/rest/action/admin/indices/RestCreateIndexAction.java

```diff
@@ -42,8 +42,8 @@
     private static final DeprecationLogger deprecationLogger = new DeprecationLogger(
         LogManager.getLogger(RestPutMappingAction.class));
     public static final String TYPES_DEPRECATION_MESSAGE = "[types removal] Specifying types in create index " +
-        "requests is deprecated. The parameter include_type_name should be provided and set to false to be " +
-        "compatible with the next major version.";
+        "requests is deprecated. To be compatible with 7.0, the mapping definition should not nested under the " +
+        "type name, and the parameter include_type_name must be provided and set to false.";
```

Comment:
>"should not nested"

Worrying, this decline in the be population.

Reply:
> "the mapping definition should not nested under the type name"

Thanks Julie, I am ok with this message as well, but may be more explicit instruction what needs to be done could be more clear, e.g using Mark's comment "To be compatible with 7.0, the type must be removed from the usual mapping -> type -> properties hierarchy, and the parameter ..."

Reply:
> Worrying, this decline in the be population.

Indeed!